src/sentry/api/endpoints/organization_profiling_profiles.py

```diff
@@ -38,10 +41,6 @@ def get_profiling_params(self, request: Request, organization: Organization) ->
 
 @region_silo_endpoint
 class OrganizationProfilingFiltersEndpoint(OrganizationProfilingBaseEndpoint):
-    publish_status = {
```

Comment:
My understanding is that we want to keep profiling private and only available from Sentry UI. Is that right? Can you please add some context on why?

In general I would keep publish_status in child Endpoint because in future someone might add an API, extend the parent class and accidentally mark that as private forever.

Reply:
That's right, all profiling endpoints are meant to be private. They were designed with only our needs in mind.